README.md

```diff
@@ -920,6 +938,251 @@ branch. Right? **Right?**
 
 So you need to merge your branch into `main`.
 
+We don't, however, want to just smash our stuff into `main`. We want to
+take advantage of our very clever colleagues and have at least one other
+person perform a code review on our proposed changes. A code review is
+essentially an online conversation where members of the team build a
+shared understanding of these changes, proposing fixes and improvements,
+and generally working together to make the code base as good as you
+collectively can.
+
+#### Create a pull request
```

Comment:
It is also possible to create the PR right from GitKraken. But maybe it's better to have them do it all in GitHub at this point.

Reply:
I thought about that, as part of a larger question of whether we do all/most of this in GitHub or in GitKraken. In the end I decided to go with GitHub, but it certainly wasn't super obvious which was the better option.